src/utils/partners/getRequiredPartners.js

```diff
@@ -0,0 +1,3 @@
+export function getFeaturedPartners(list, featured) {
```

Comment:
Do you really think that this function should be in utils folder? I think that it's a very specific thing about the partner's component... Let's move it